SYCL/ESIMD/ext_math.cpp

```diff
@@ -61,10 +71,19 @@ template <MathOp Op> float HostMathFunc(float X);
     }                                                                          \
   }
 
-DEFINE_OP(sin, sin);
-DEFINE_OP(cos, cos);
-DEFINE_OP(exp, exp);
-DEFINE_OP(log, log);
+#define DEFINE_OP_REUSE_SCALAR(Op, HostOp)                                     \
```

Comment:
I think it is worth to clearly state that this is std sycl:
```suggestion
#define DEFINE_SIMD_OVERLOADED_STD_SYCL_OP(Op, HostOp)                                     \
```

Reply:
Done